Eclipse_Neon_4.6.2_PHRX_Formatter.xml

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We should define a common project style, but not in an eclipse-specific format. Something like checkstyle would be used. My proposal would be to leave the style changes to another PR.